schema.graphql

```diff
@@ -68,3 +68,15 @@ type EscrowEvent @entity {
   gasUsed: BigInt!
   gasPrice: BigInt!
 }
+
+type SingleRequestProxyDeployment @entity {
+  id: ID!
+  proxyAddress: Bytes!
+  payee: Bytes!
+  tokenAddress: Bytes
+  paymentReference: Bytes!
+  proxyType: String! # "Ethereum" or "ERC20"
+  block: Int!
+  timestamp: Int!
+  txHash: Bytes!
+}
```

Comment:
We don't need the `created_at` field because we already have the `timestamp` field. :no_good: 

We don't need the `network` field because a subgraph is chain-specific. :no_good: 

I agree that an enum for proxyType makes sense :+1: